src/ToolTipsFunctions.ts

```diff
@@ -1,6 +1,105 @@
+import * as fs from 'fs';
 import * as vscode from 'vscode';
 import * as path from 'path';
-import { ALObject } from './ALObject';
+import { Settings, Setting } from "./Settings";
+import { ALObject, ControlType, ObjectProperty } from './ALObject';
+import * as WorkspaceFunctions from './WorkspaceFunctions';
+
+export async function GenerateMarkDownDocs() {
```

Comment:
Suggestion: If this is to be extended in the future a MarkDownDocument class would be in order? Perhaps overkill but it would look nice ;) Also it would help with formatting such as blank lines before and after Headers, tables and such.

Reply:
You can refactor that when I'm done if you want to? :)